third_party/mbedtls.BUILD

```diff
@@ -0,0 +1,8 @@
+cc_library(
+    name = "mbedtls2.16",
```

Comment:
Why does it have suffix `2.16`?

Reply:
The entangled has a same library name .

Reply:
You have to explain within comments along with git commit messages to address the potential conflicts.